addon/lib/field.js

```diff
@@ -400,10 +400,18 @@ export default Base.extend({
         getWithDefault(this, "fieldset.field.hidden", false) ||
         (this.hiddenDependencies.length &&
           this.hiddenDependencies.every(fieldIsHidden)) ||
-        this.document.jexl.evalSync(
-          this.question.isHidden,
-          this.document.jexlContext
-        )
+        this.document.jexl.evalSync(this.question.isHidden, {
+          ...this.document.jexlContext,
+          info: {
```

Comment:
As discussed, the `info` context also needs to be added to the optional property (line 429)

Reply:
Added the computed property `jexlContext` which is used in `hidden` and `optional`.